mne/io/ctf/ctf.py

```diff
@@ -165,6 +165,22 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                 _mult_cal_one(data_view, this_data, idx, cals, mult)
                 offset += n_read
 
+    def _clean_names(self):
+        """Clean up CTF suffixes from channel names.
+
+        Notes
+        -----
+        .. versionadded:: 0.16.0
+        """
+
+        mapping = dict(zip(self.ch_names, _clean_names(self.ch_names)))
+
+        super(RawCTF, self).rename_channels(mapping)
+
+        for comp in self.info['comps']:
+            for key in ('row_names', 'col_names'):
+                comp['data'][key] = _clean_names(comp['data'][key])
+
 
```

Comment:
@OlehKSS can you add a parameter to the CTF constructor that for now defaults to False but on True calls this method?

Reply:
If this parameter will be True and in case _clean_names method is called, cleaning of the compensation channels should be triggered after cleaning of the main channels, am I correct? Or should I add this parameter to the constructor and if it is True constructor should call _clean_names method?

Reply:
@OlehKSS The idea is to call ```self._clean_names``` on say ```clean_names=True``` while letting it default to False. Only that parameter would need to be documented since the method you added will remain private. Is it clearer?

Reply:
@dengemann Yes, thanks

Reply:
@dengemann  I have updated constructor according to your description